doc/admx/en-US/DesktopAppInstaller.adml

```diff
@@ -122,20 +122,24 @@ If you disable this setting, users will not be able to use the Windows Package M
 If you disable or do not configure this setting, no proxy will be used by default.
 
 If you enable this setting, the specified proxy will be used by default.</string>
+      <string id="EnableWindowsPackageManagerAllowedSecurityZones">Enable App Installer Allowed Zones for the Windows Package Manager</string>
+      <string id="EnableWindowsPackageManagerAllowedSecurityZonesExplanation"></string>
+      <string id="EnableWindowsPackageManagerSmartScreenCheck">Enable Microsoft SmartScreen checks for the Windows Package Manager</string>
+      <string id="EnableWindowsPackageManagerSmartScreenCheckExplanation"></string>

```

Comment:
Adding @RDMacLachlan to help with the strings for the new group policies in the ADML